includes/class-filesystem-direct.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * The Class for WordPress Direct Filesystem with optimized read and write routines.
+ *
+ * @package aspire-update
+ */
+
+namespace AspireUpdate;
+
+/**
+ * The Class for WordPress Direct Filesystem with optimized read and write routines.
+ */
+class Filesystem_Direct extends \WP_Filesystem_Direct {
+	/**
+	 * Write contents to a file with additional modes.
+	 *
+	 * @param string $file The path to the file.
+	 * @param string $contents The content to write.
+	 * @param int|false $mode     Optional. The file permissions as octal number, usually 0644.
+	 *                            Default false.
+	 * @param string $write_mode The write mode:
+	 *                     'w'  - Overwrite the file (default).
+	 *                     'a'  - Append to the file.
+	 *                     'x'  - Create a new file and write, fail if the file exists.
+	 *                     'c'  - Open the file for writing, but do not truncate.
```

Comment:
Can we align the `@param` names and descriptions for easier reading?
```suggestion
	 * @param string    $file       The path to the file.
	 * @param string    $contents   The content to write.
	 * @param int|false $mode       Optional. The file permissions as octal number, usually 0644.
	 *                              Default false.
	 * @param string    $write_mode The write mode:
	 *                              'w'  - Overwrite the file (default).
	 *                              'a'  - Append to the file.
	 *                              'x'  - Create a new file and write, fail if the file exists.
	 *                              'c'  - Open the file for writing, but do not truncate.
```

Reply:
Format needs to be run on the code